packages/test-utils/src/create-dom-event.js

```diff
@@ -24,6 +24,19 @@ const modifiers = {
   pagedown: 34
 }
 
+function getOptions(modifier, { bubbles, cancelable }, options) {
+  const keyCode = modifiers[modifier] || options.keyCode || options.code
+
+  const derivedOptions = { keyCode, code: keyCode }
```

Comment:
I'm sure I might be missing something here, but why do we need to create the `derivedOptions` variable instead of directly adding them to the return statement?

```js
  return {
    ...options, // What the user passed in as the second argument to #trigger
    bubbles,
    cancelable,
    keyCode,
    code: keyCode
  }
```

Reply:
So that it’s very clear what order wins in the override/merge.  We’re gonna have more props in the future. The extra object acts as a logical grouping and communicates why certain properties win over others.

Reply:
I’m happy to in-line it and use a comment block as a delimiter if that’s preferred